Doc/library/importlib.rst

```diff
@@ -420,30 +420,34 @@ ABC hierarchy::
         reloaded):
 
         - :attr:`__name__`
-            The name of the module.
+            The module's fully-qualified name.
 
         - :attr:`__file__`
-            The path to where the module data is stored (not set for built-in
-            modules).
+            The location of the module's source code.
```

Comment:
You are right, the "location" can be interpreted as the parent directory, I'll keep the "path".

Reply:
Hmm, you have a point.  It matters for custom importers.  However, it would be worth being clear about the common case:
```suggestion
            The location (usually filename) of the module's source code.
```

Reply:
I'm going to bring this up in a new review.